apps/src/applab/ViewDataButton.jsx

```diff
@@ -0,0 +1,49 @@
+/** @file Button that opens the data browser */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var styles = require('./PlaySpaceHeaderStyles');
+
+/**
+ * The button above the visualization that opens the data browser.
+ * @type {function}
+ */
+var ViewDataButton = React.createClass({
+  propTypes: {
+    onClick: React.PropTypes.func.isRequired
+  },
+
+  render: function () {
+
+    var showDataButtonStyle = $.extend(
+        {
```

Comment:
Yeah, reading our style guide again, I think you probably do meet it. I think that maybe I just don't like this particular style :-P  The only time that I like a four space indent is when the line continuation precedes a block, and I want to distinguish the two different lines, for example
```
// bad
module.exports.fairlyLongFunction = function fairlyLongFunction(argumentNumber1,
  argumentNumberTwo, argumentNumber3) {
  console.log("I'm aligned with the continuation");
}

//good
module.exports.fairlyLongFunction = function fairlyLongFunction(argumentNumber1,
    argumentNumberTwo, argumentNumber3) {
  console.log("I'm not aligned with the continuation");
}
```
Anywhere else, it feels like we're just wasting horizontal space.